### PR TITLE
fix(config): param 9 for STEINEL devices, rework to templates

### PR DESCRIPTION
**Pull request by nechry**
PR description here
fix(config): param 9 for STEINEL devices

### CODE REVIEW

**Comment by nechry** [comment]
@zwave-js-bot fix lint

**Review by blhoward2 — CHANGES_REQUESTED** [review]
Thanks for helping out! Your changes are great so far, but we generally try to avoid descriptions where we can. Could you please take another look and remove them if unneeded? If necessary, sometimes modifying the label (while keeping them short) makes them redundant. If something must truly be described that's fine but they clutter up the UIs.

**Comment by nechry** [comment]
Hello, I simply used the labels and descriptions already present for the xled_home_2 and applied to the other models as they share the same configuration base.
I will look to simplify them or as you recommended to remove them.

I fully understand your point, I maintained for more than 8 years the configuration files for the openzwave library

**Comment by AlCalzone** [comment]
Hey @nechry nice to see you at the other side of Z-Wave land.

We like to take the chance to clean up files (many of which were auto-imported from various sources at some point) when somebody touches them.

**Comment by nechry** [comment]
@AlCalzone  Well noted, I think I'm getting more and more involved here.

**Comment by nechry** [comment]
@blhoward2 done, always happy to contribute

File: packages/config/config/devices/0x0271/motionswitch_led.json [hunk 1]
```diff
@@ -75,13 +75,40 @@
 			"defaultValue": 1
 		},
 		{
-			"#": "9",
-			"label": "SLAVE_MODE",
-			"description": "Disable local control",
+			"#": "9[0x01]",
+			"label": "Gateway Check",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0271/motionswitch_led.json]: @nechry Do you have a manual which shows that this one has no "External Control" mode? I only found [this](http://manuals-backend.z-wave.info/make.php?lang=en&sku=40111665101&cert=ZC10-18076193) (near the bottom) and it looks like param 9 is the same as for the other devices.

**Review by AlCalzone — COMMENTED** [review]
I'm going to take over and template this and the other parameters too. Just one question...

File: packages/config/config/devices/0x0271/motionswitch_led.json (hunk 1 above)
Comment (nechry) [reply packages/config/config/devices/0x0271/motionswitch_led.json]: I found this: [one](https://manuals-backend.z-wave.info/make.php?lang=en&sku=STEEMSWITCH&cert=ZC10-18076193) This is the only model I don't own.
your document seems more recent to me, so I prefer to play safe and add the bitset.

Comment (AlCalzone) [reply packages/config/config/devices/0x0271/motionswitch_led.json]: Thanks, I'll go with the bitset then. Even on the one you linked it looks a bit like it is supposed to be a bitset but they entered the values weirdly.

Comment (nechry) [reply packages/config/config/devices/0x0271/motionswitch_led.json]: in fact like bit 2 (LED is permanently on) override bit 0 and 1 it is true that we could simplify with a list. Again I relied on the initial config for the refactoring.

Comment (nechry) [reply packages/config/config/devices/0x0271/motionswitch_led.json]: @AlCalzone do you prefer I convert to list with selection:

```json
{
  "#": "9",
  "label": "Slave mode",
  "description": "",
  "valueSize": 1,
  "minValue": 0,
  "maxValue": 4,
  "defaultValue": 2,
  "unsigned": true,
  "allowManualEntry": false,
  "options": [
  {
	  "label": "Normal mode",
	  "value": 0
  },
  {
	  "label": "Slave mode",
	  "value": 1
  },
  {
	  "label": "Normal mode with checking lifeline",
	  "value": 2
  },
  {
	  "label": "Slave mode with checking lifeline",
	  "value": 3
  },
  {
	  "label": "LED is permanently on",
	  "value": 4
  }
  ]
}
```

Comment (AlCalzone) [reply packages/config/config/devices/0x0271/motionswitch_led.json]: I reworked most of the files to templates, let me just copy that. It's probably easier to understand than the bitset.

Comment (nechry) [reply packages/config/config/devices/0x0271/motionswitch_led.json]: great, yes at the end a list even if behind it's bit sets it's even more user-friendly

**Review by AlCalzone — APPROVED** [review]

**Comment by blhoward2** [comment]
> @AlCalzone  Well noted, I think I'm getting more and more involved here.

As Al said, welcome to the other side! We can always use more help with the device files. If you're going to contribute more, in the docs there is a style guide for device files and sections on templating parameters and conditional parameters. That should take out some of the mystery of where our requested changes frequently come from, to avoid less back and forth.

**Pull request merged**